custom/enikshay/nikshay_datamigration/models.py

```diff
@@ -285,8 +285,8 @@ class Outcome(models.Model):
     )
     InitiatedDate = models.CharField(max_length=255, null=True)  # dates, None, and NULL
     userName = models.CharField(max_length=255)
-    loginDate = models.DateTimeField()
-    OutcomeDate1 = models.CharField(max_length=255)  # datetimes and NULL
+    # loginDate = models.DateTimeField()
+    # OutcomeDate1 = models.CharField(max_length=255)  # datetimes and NULL
```

Comment:
I don't follow.  https://github.com/dimagi/commcare-hq/pull/14772/files#diff-9b2457f77a1639e5734852448332815b should remove them, right?